pep-0646.rst

```diff
@@ -867,6 +867,7 @@ After:
 
     star_etc:
     | '*' param_no_default param_maybe_default* [kwds]
+    # This is the new alternative. Other alternatives are unchanged.
```

Comment:
With whole-line comments it's always a bit ambiguous where they point. Maybe just add `# New` to the end of the new alternative?

Reply:
Done.